src/renderer/state.ts

```diff
@@ -716,7 +723,9 @@ export class AppState {
 
     for (const id of ALL_MOSAICS) {
```

Comment:
This is the only place in `src/` where `ALL_ MOSAICS` is used and it could be replaced by

```
for (const id of currentlyVisible) {
  if (!visible.includes(id)) {
    ...
  }
}

for (const id of visible)) {
  if (!curentlyVisible.includes(id)) {
    ...
  }
}
```

IDK how this would play in the tests, but this looks like low-hanging fruit for removing one piece of cruft